kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -400,6 +400,37 @@ public abstract class MobiComConversationFragment extends Fragment implements Vi
     protected String autoSuggestUrl;
     private Set<Message> botDelayMessageList;
     private List<String> localTeams;
+    private static final String KMEVENT = "KM_EVENT";
+    private static final String QUICK_REPLY_CLICK = "QUICKREPLY_CLICK";
+    private static final String SKIP_BOT = "skipBot";
+    private static final String CATEGORY = "category";
+    private static final String KM_QUICK_REPLY = "KM_EVENT_QUICKREPLY_CLICK";
+    private static final String BOT = "bot";
+    private static final String HIDDEN = "HIDDEN";
+    private static final String VIDEO = "video";
+    private static final String AUDIO = "audio";
+    private static final String CAMERA = "camera";
+    private static final String text_plain = "text/plain";
+    private static final String text_card = "text/x-vcard";
+    private static final String IMAGE = "image";
+    private static final String GALLERY = "gallery";
+    private static final String LOCATION = "location";
+    private static final String CONTACT = "contact";
+    private static final String NAME = "[name]";
+    private static final String IMPLEMENT_INTERFACE_COMMUNICATOR = " must implement interfaceDataCommunicator";
+    private static final String IS_DONE_WITH_CLICKING = "isDoneWithClicking";
+    private static final String APPLOZIC_TEMPLATE_MSG = "com.applozic.mobicomkit.TemplateMessage";
+    private static final String TEMPLATE_MSG = "templateMessage";
+    private static final String APPLI_JSON = "application/json";
+
+    /*
+    autoSuggestionMetadata.put("KM_EVENT", "QUICKREPLY_CLICK");
+                            autoSuggestionMetadata.put("skipBot", "true");
+                            autoSuggestionMetadata.put("category", "HIDDEN");
+                            sendMessage("KM_EVENT_QUICKREPLY_CLICK", autoSuggestionMetadata, null, null, Message.ContentType.DEFAULT.getValue());
+    * */
+
```

Comment:
What is the reason of adding this code and why it's commented ?

Reply:
Removed this comment, was just added at the time of declaring constants.